docs/posts/ibis-version-9.0.0-release/index.qmd

```diff
@@ -0,0 +1,351 @@
+---
+title: "Ibis 9.0: SQLGlot-ification"
+author: "Ibis team"
+date: "2024-04-18"
+categories:
+    - release
+    - blog
+---
+
+## Overview
+
+Ibis 9.0 completes ["the big
+refactor"](../../posts/roadmap-2024-H1/index.qmd#the-big-refactor), completing
+the transition from SQLAlchemy to [SQLGlot](https://github.com/tobymao/sqlglot)
+and drastically simplifying the codebase. This is a big step toward stabilized
+internals and allows us to more easily add new features and backends going
+forward.
+
+Ibis 9.0 also adds new functionality, fixes many bugs, and welcomes a new
+committer to the project!
+
+### New committer
```

Comment:
@ncclementi is also a new committer: https://github.com/ibis-project/governance/discussions/6

Reply:
I didn't realize this post was drafted well before that happened. Anyway.